crates/cxx-qt-lib-headers/include/gui/qguiapplication.h

```diff
@@ -18,5 +19,11 @@ namespace cxxqtlib1 {
 ::std::unique_ptr<QGuiApplication>
 qguiapplicationNew(const QVector<QByteArray>& args);
 
+void
+qapplicationSetFont(QGuiApplication& app, const QFont& font)
```

Comment:
implementation should be in .cpp file as it's not a template, but also see comment below probably can remove all the C++ code related to font() / setFont()